model/pom.xml

```diff
@@ -4,48 +4,70 @@
 	<parent>
 		<groupId>com.ft.unifiedContentModel</groupId>
 		<artifactId>ft-unifiedContentModel-parent</artifactId>
-		<version>0.41.0</version>
+		<version>1.0.0</version>
 	</parent>
 	<artifactId>ft-unifiedContentModel-model</artifactId>
 	<packaging>jar</packaging>
 	<name>${project.artifactId}</name>
 	<description>Unified Content Model - model implementation</description>
 	<dependencies>
-		<dependency>
-			<groupId>javax.servlet</groupId>
-			<artifactId>servlet-api</artifactId>
-			<scope>provided</scope>
-		</dependency>
 		<dependency>
 			<groupId>${project.groupId}</groupId>
 			<artifactId>ft-unifiedContentModel-core</artifactId>
 			<version>${project.version}</version>
+			<scope>compile</scope>
 		</dependency>
 		<dependency>
-			<groupId>org.codehaus.jackson</groupId>
-			<artifactId>jackson-mapper-asl</artifactId>
+			<groupId>javax.xml.bind</groupId>
+			<artifactId>jaxb-api</artifactId>
 		</dependency>
 		<dependency>
-			<groupId>org.codehaus.jackson</groupId>
-			<artifactId>jackson-xc</artifactId>
+			<groupId>javax.annotation</groupId>
+			<artifactId>javax.annotation-api</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>org.springframework</groupId>
+			<artifactId>spring-core</artifactId>
```

Comment:
I consider `joda-time` as deprecate as of java 8 because now java have a good support for Time in the standard, but I didn't remove this because is not a blocking change for the search API port project, maybe we can create a ticket and assign a priority for  this @mitagg ?

`spring` is a dependency here, [for example](https://github.com/Financial-Times/unified-content-model/blob/master/model/src/main/java/com/ft/api/ucm/model/v1/aspect/ConcreteFieldResolutionPolicy.java#L3).

Reply:
Agreed, we can make improvements and modernizations after we are done with the search API port upgrade. @denisacostaq can you ask Bogdana to assist you in creating a ticket for this?